packages/amplify_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 0.3.0-rc.4 (2022-01-12)
```

Comment:
Should we announce the breaking change about
```
import 'package:amplify_flutter/amplify.dart';
to 
import 'package:amplify_flutter/amplify_flutter.dart';
```
here and probably a migration instruction?

Reply:
I will add this along with change logs for the rest of the packages

Reply:
Updated